src/org/jitsi/impl/neomedia/codec/video/vp8/DePacketizer.java

```diff
@@ -542,20 +542,91 @@ public static int getSize(byte[] input, int offset)
         }
 
         /**
+         * Determines whether the VP8 payload specified in the buffer that is
+         * passed as an argument has a picture ID or not.
+         *
+         * @param buf the byte buffer that contains the VP8 payload.
+         * @param off the offset in the byte buffer where the VP8 payload
+         * starts.
+         * @param len the length of the VP8 payload in the byte buffer.
+         * @return true if the VP8 payload contains a picture ID, false
+         * otherwise.
+         */
+        public static boolean hasPictureId(byte[] buf, int off, int len)
+        {
+            if (!isValid(buf, off))
+                return false;
+
+            if ((buf[off] & X_BIT) == 0
+                || (buf[off+1] & I_BIT) == 0)
+                return false;
```

Comment:
I would simplify to "return isValid(buf,off) && (buf[off] & X_BIT != 0) && (buf[off+1] & I_BIT) != 0)"